sites/game-client/ui/modal.mjs

```diff
@@ -112,7 +112,8 @@ class Modal {
    * If the modal has ToolbarButton, it should show the modal
    * when the button is clicked.
    */
-  onClickToolbarButton() {
+  onClickToggleButton() {
+    // XXX: The behavior is not matched its name.
```

Comment:
If I remember correctly, this is overridden by other users of this class, we might have to update them as well if we're to rename this.